style.css

```diff
@@ -0,0 +1,1131 @@
+html {
+	box-sizing: border-box;
+}
+
+*, *:before, *:after {
+	box-sizing: inherit;
+}
+	
+body {
+	min-height: 100vh;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/GerKateAxion/pullRequest?prid=5991795)